packages/image_picker/image_picker_linux/lib/image_picker_linux.dart

```diff
@@ -0,0 +1,151 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file_selector_linux/file_selector_linux.dart';
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+import 'package:flutter/foundation.dart';
+import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
+
+/// The Linux implementation of [ImagePickerPlatform].
+///
+/// This class implements the `package:image_picker` functionality for
+/// Linux.
+class ImagePickerLinux extends CameraDelegatingImagePickerPlatform {
+  /// Constructs a platform implementation.
+  ImagePickerLinux();
```

Comment:
Could this be const?

Reply:
We don't generally make our platform implementation classes const. I don't think there's much benefit to it, and it would lock them into being const forever without a breaking change. Is there any real upside to it?